lib/chat/chunked_files_multikey.ex

```diff
@@ -0,0 +1,47 @@
+defmodule Chat.ChunkedFilesMultikey do
+  @moduledoc false
+
+  alias Chat.Db
+  alias Chat.Utils
+
+  @chunk_group_size 1024 * 1024 * 1024
```

Comment:
https://github.com/Buckitup-chat/chat/blob/main/lib/chat/chunked_files.ex#L86

it would be `1000 * 1024 * 1024`